migrate.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS `vehicle_financing` (
+    `id` int(11) NOT NULL AUTO_INCREMENT,

```

Comment:
Shouldn't this id be a foreign key of the vehicle id itself? I wouldn't expect us to need to support multiple finance entries per vehicle

Reply:
yeah that's a better idea 👍🏻